backend/src/routes-admin.ts

```diff
@@ -60,8 +60,8 @@ router.get('/orders/:orderid/tickets/send', function(req: Request, res: Response
   order.sendTickets(parseInt(req.params.orderid)).then(() => res.sendStatus(200));
 });
 
-router.get('/orders/:orderid/tickets/:ticketid/delete', function(req: Request, res: Response) {
-  order.removeTicket(parseInt(req.params.orderid), parseInt(req.params.ticketid)).then(ok(res), err(res));
+router.get('/orders/:orderid/tickets/:ticketid/:tickethash/delete', function(req: Request, res: Response) {
```

Comment:
This was changed earlier from DELETE -> GET due to mod_rewrite, but could we switch again to POST? The reason being that there are some differences in browser impl in get vs post, whereas post vs delete isn't that big a difference.